setup.py

```diff
@@ -65,8 +65,8 @@
         "redis~=3.5.3",
         "resolwe-runtime-utils~=3.0.0",
         "shellescape~=3.8.1",
-        "Sphinx>=1.5.1, <1.7.0",
         "beautifulsoup4~=4.9.3"
+        "Sphinx",
```

Comment:
Limit the version of the Sphinx.

Reply:
OK, will pin it to the current stable version.